README.md

```diff
@@ -150,6 +150,8 @@ Available configuration options for *both* subscriber and publisher applications
 * `middleware`: A chain of middleware that messages must go through when sent or received.
   Please refer to the [Middleware](#middleware) section for more details regarding this
   option.
+* `aws_options_overrides`: A key/value hash of option overrides passed through to the
```

Comment:
:neckbeard: Maybe we can also say `option overrides and additions` since we can pass any supported param now and not just override the few as base options.

Reply:
Good point @derekclee , I've added that to this description so it's more clear that it's not just overrides